### PR TITLE
deployment(docker/nginx): addressing the 403's -- fonts and the favicon

### LINKED ISSUE #770: [Deployment] fonts and favicon not getting served (403/404 responses)
Issue is potentially in the updated `nginx.conf`

## Current behavior
- Beta staging site (not EKS) has the issue - 403 response on fonts and favicon

## Screenshots

Screenshot 2024-06-26 at 8 50 28 AM

### PR DESCRIPTION
**Pull request by shindigira**
closes #770 

## Changes
- deployment(nginx): added routes to open fonts and the favicon to routes / and /static/assets/

## How to Test
- Ensure it builds with `yarn docker-build-and-run`
- Test in the dev or staging, ensure the fonts and favicons no longer get 403's or 404's

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -45,6 +45,11 @@ RUN sed -i '/Faithfully yours/d' /usr/share/nginx/html/50x.html && \
     touch /run/nginx.pid && \
     touch /var/run/nginx.pid && \
     chown -R $NGINX_USER:$NGINX_USER /etc/nginx /run/nginx.pid /var/cache/nginx/ /var/run/nginx.pid /usr/share/nginx/html/index.html /usr/share/nginx/html/import-meta-env-alpine /usr/share/nginx/html/nginx-entrypoint.sh /usr/share/nginx/html/.env.example
+    
+  # Adjust file permissions for asset files (favicon, png's, fonts, etc)
+  RUN find /usr/share/nginx/html -type d | xargs chmod 755 && \ find /usr/share/nginx/html -type f | xargs chmod 644
```
Comment (lchen-2101) [inline Dockerfile:50]: if this step helps, maybe adding to the Dockerfile would be a better place for it; towards end of the dockerfile there is a chown command, so possibly add this dir in there as well.

**Comment by billhimmelsbach** [comment]
Heyo! This latest one isn't building for me?
Screenshot 2024-06-26 at 4 09 18 PM

**Comment by billhimmelsbach** [comment]
Looking good on the preview/dev-pub site!
Screenshot 2024-06-27 at 11 19 58 AM
Screenshot 2024-06-27 at 11 19 50 AM

**Review by billhimmelsbach — DISMISSED** [review]
Looking good on the preview/dev-pub site @shindigira!
Screenshot 2024-06-27 at 11 19 58 AM
Screenshot 2024-06-27 at 11 19 50 AM

File: Dockerfile [hunk 2]
```diff
@@ -37,16 +37,23 @@ COPY nginx/nginx.conf /etc/nginx/templates/nginx.conf.template
...
+
 # Security Basline - The `sed` was added to meet requirement 17
 RUN sed -i '/Faithfully yours/d' /usr/share/nginx/html/50x.html && \
     addgroup -S $NGINX_USER && \
     adduser -S $NGINX_USER -G $NGINX_USER && \
     # We need to come back and reconcile the multiple pids.
     touch /run/nginx.pid && \
     touch /var/run/nginx.pid && \
-    chown -R $NGINX_USER:$NGINX_USER /etc/nginx /run/nginx.pid /var/cache/nginx/ /var/run/nginx.pid /usr/share/nginx/html/index.html /usr/share/nginx/html/import-meta-env-alpine /usr/share/nginx/html/nginx-entrypoint.sh /usr/share/nginx/html/.env.example
+    touch /var/run/nginx.pid && \
```
Comment (hkeeler) [inline Dockerfile:48]: This command is already performed in the line above.

**Review by hkeeler — COMMENTED** [review]
Just the one dupe command, otherwise LGTM.

**Review by billhimmelsbach — APPROVED** [review]
It's working well! 👍

**Pull request merged**